microsoft-defender/src/openbas_microsoft_defender.py

```diff
@@ -226,17 +238,22 @@ async def _process_alerts(self, graph_client):
         self.helper.collector_logger.info(
             "Found " + str(len(expectations)) + " expectations waiting to be matched"
         )
+
+        if not any(expectations):
+            self.helper.collector_logger.info("No work: skipping iteration.")
```

Comment:
**suggestion:**
self.helper.collector_logger.info("No expectations found: skipping iteration.")
It's more readable ?